src/source/core/Tasks.bs

```diff
@@ -112,6 +112,10 @@ namespace mc.tasks
       mc.tasks.observeNodeField(task, fieldName, callback, sendMode, true, scope)
     end if
 
+    if asBoolean(globalObj.isTrackingTaskUsage)
```

Comment:
I propose we create a new maestroConfig object, so that we don't fill up global with flags. let's move any global maestro flags to this.

`globalObj.maestroConfig.isTrackingTaskUsage`